migrator.sh

```diff
@@ -367,7 +392,7 @@ query_source_images() {
     for i in ${REPO_LIST}
     do
       # get tags for repo
-      IMAGE_TAGS=$(curl ${INSECURE_CURL} -sf -H "Authorization: JWT ${TOKEN}" https://hub.docker.com/v2/repositories/${NAMESPACE}/${i}/tags/?page_size=100000 | jq -r '.results|.[]|.name') || catch_error "curl => API failure"
+      IMAGE_TAGS=$(curl ${INSECURE_CURL} -sf -H "Authorization: JWT ${TOKEN}" https://hub.docker.com/v2/repositories/${NAMESPACE}/${i}/tags/?page_size=100000 | jq -r '.results|.[]|.name') || echo "Hey man, this didn't work: ${NAMESPACE}/${i}, ya dig?"
```

Comment:
This no longer catches the error; would be good if this could be optional to skip or fail like the other comment.  It would be nice if the error was descriptive, switching this:
`echo "Hey man, this didn't work: ${NAMESPACE}/${i}, ya dig?"`

to something more specific to the tune of curl to the API failed to retrieve tags. with the namespace and repo.  I definitely like that it tells you what failed though.